File: csv2json.js
'use strict';

const csv2json = require('csv2json');
const fs = require('fs');
const path = require('path');

const CSV_PATH = path.join('T:\\ProgramFilesT\\pleiades\\workspace\\node225', 'nt1570.csv');

fs.createReadStream(CSV_PATH)
    .pipe(csv2json({
        // Defaults to comma.
        //separator: ';'
    }))
    .pipe(fs.createWriteStream('./json/nt1570.json'));

File: old/csv2json.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
import pandas as pd


def run():
    folder_path = "T:\\ProgramFilesT\\pleiades\\workspace\\node225"
    csv_path = os.path.join(folder_path, "nt1570.csv")

    df = pd.read_csv(csv_path)
    df.to_json("./json/nt1570.json", orient="records")


if __name__ == "__main__":
    run()
